_chapters/js/dependencies.md

```diff
@@ -22,6 +22,29 @@ It's not needed and should be phased out and replaced by npm. More information c
 
 npm installs its dependencies in the `node_modules` directory. Common conventions dictate that `node_modules` should be excluded from source control by adding it to your project's `.gitignore`, primarily because Node.js-friendly environments (such as 18F's deployment service, [Cloud Foundry], and other such as [Heroku]) recognize the existence of `package.json` and automatically install dependencies as needed.
 
+### Publishing
+#### Scoping a package to the 18F npm org
+18f has an [npm organization](https://www.npmjs.com/org/18f) that is meant to organize permissions and packages related to 18f. As an 18F developer, when publishing a package, you have the choice whether to scope a package to the 18F org or not. Scoped package will always be prefixed with `@18f/` before their package name and can have their permissions managed by people in the org. More information about scoped packages can be found on the [npm documentation](https://docs.npmjs.com/misc/scope).
+
+##### Guidance on when to scope a package or not
+- A package *should not* be scoped to 18f if consumers of the package (either gov or non-gov) do not need to know about 18f in order to use the package.
+  - Example: The Draft US Web Design Standards are used by many entities outside of 18f and government. When user's go to install the WDS, the don't need to know anything about 18f.
+- A package *should* be scoped to 18f if it's use cases falls main inside 18f.
```

Comment:
"A package *should* be scoped to 18F if its use cases fall mainly inside of 18F."